src/components/Cards/Cards.tsx

```diff
@@ -16,8 +30,10 @@ export const Cards: React.FC<CardsProps> = ({ posts }) => {
 
   return (
     <section className='cards-container'>
+      {loading && <p>Loading...</p>}
+      {error && <p>Error:</p>}
```

Comment:
maybe a more user friendly error?

Reply:
We'll have to figure out what this looks like, I haven't seen what a graphQL error is yet